### PR TITLE
Registry August update

### PR DESCRIPTION
**Pull request by ingenieroariel**
- Multi-catalog support
 - OpenSearch support
 - Improved CSW harvesting
 - Settings were all renamed to start with REGISTRY, there is no need for custom templates at the exchange level (only modify base if you want to change the theme).

 - A default catalog fixture called 'hypermap' is provided if you don't want users to have to create them manually.

 - Layers are now checked every 24 hours

 - Layers are indexed in bulk, cache is cleared every minute

 - CATALOGLIST is deprecated, catalog list is now fetched from /registry/api/catalogs

```
Added fixtures and removing tests in makefile
Using django-mapproxy from pypi
Django deprecated initial data in 1.7
Look for SEARCH_PRECISION env var
Make REGISTRY_CHECK_PERIOD configurable via env vars
Move to DATABASE_URL
migrate account before migrate all
Set REGISTRY_CHECK_PERIOD to 5 minutes for development
Updated maploom version with fixes from Terranodo
Correct nginx image in override
Added default catalog fixture
Make sure pycsw and registry are the right version
Indexing correctly and showing in maploom
remove trailing slash from site url
move templates to take hypermap/registry
Use django-registry 0.2.8:
Upgrade maploom, registry and pycsw in development
roll back celery schedule changes
Switch to pypi geonode version to avoid conflicts with pycsw
Bump hypermap and django maploom
Added registryUrl to UI
fix menu links
```

### CODE REVIEW

**Comment by ingenieroariel** [comment]
Seems like master has moved on since this PR was created - I'll update it tomorrow.

File: requirements.txt [hunk 1]
```diff
@@ -20,7 +20,11 @@ gunicorn==19.4.5
...
-git+git://github.com/geopython/pycsw.git@1525e70d36e363a856f03988ceb1598ba00d87f0#egg=pycsw
-pdb
+djmp==0.2.8
+django-registry==0.2.17
+# To be replaced by pycsw 2.0.1 when it is out.
+pycsw-hypermap==2.0.1
+# To be replaced by django-exchange-maploom when it includes the latest in the following branch:
+# git+git://github.com/terranodo/django-terranodo-maploom.git@production#egg=django-maploom
+# Latest hash is bfed4deb1f0f7ce11c9151fd09884d11cabd529a
+django-terranodo-maploom==1.5.6
```
Comment (davisc) [inline requirements.txt]: @ingenieroariel should we switch to this repo/branch or use the boundless repo/branch. Trying to figure out if the boundless repo/branch is in need of changes?

Comment (ingenieroariel) [inline requirements.txt]: Since you merged https://github.com/boundlessgeo/MapLoom/pull/27 it is not needed anymore. I'll update.

Comment (ingenieroariel) [inline requirements.txt]: Actually, @davisc can you push a new release to https://github.com/boundlessgeo/django-exchange-maploom ?

Comment (davisc) [inline requirements.txt]: Will do.

Comment (ingenieroariel) [inline requirements.txt]: Okay - I am rebasing with master and squashing.

File: exchange/templates/base.html [hunk 2]
```diff
@@ -205,11 +205,11 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Menu" %}</h4>
               <li class="modal-divider"></li>
               <li><a href="{{ GEOSERVER_BASE_URL }}"><i class="fa fa-gears"></i> {% trans "GeoServer" %}</a></li>
               {% if REGISTRY %}
-              <li><a href="/registry"><i class="fa fa-server"></i> {% trans "Registry" %}</a></li>
+              <li><a href="/registry/"><i class="fa fa-server"></i> {% trans "Registry" %}</a></li>
               {% endif %}
               {% endif %}
               {% if REGISTRY %}
-              <li><a href="/registry/search/csw?service=CSW&version=2.0.2&request=GetCapabilities" target="_blank"><i class="fa fa-globe"></i> {% trans "Registry CSW" %}</a></li>
+              <li><a href="/registry/hypermap/csw/?service=CSW&version=2.0.2&request=GetCapabilities" target="_blank"><i class="fa fa-globe"></i> {% trans "Registry CSW" %}</a></li>
```
Comment (davisc) [inline exchange/templates/base.html]: Is this correct? changing `search` to `hypermap`

Comment (ingenieroariel) [inline exchange/templates/base.html]: You are right, there is no default Registry pycsw anymore but one pycsw per catalogue. This line should be removed altogether. Updating the PR.

**Pull request merged**